examples/llama7b_w4a16_quantization.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>definitely should update to simplify after the UX fixes - ideally we don't need to expose too much about recipes/modifiers here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuralmagic/sparseml/pull/2285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yup, once the UX PRs get merged in I'll update the examples